spec/services/fake_users_and_child_data_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe FakeUsersAndChildData do
+  let(:row) do
+    {
+      "first_name" => "Joe",
+      "last_name" => "Bloggs",
+      "email" => "joe@bloggs.com",
+      "child_first_name" => "Tom",
+      "child_last_name" => "Mulligan",
+    }
+  end
+  subject { described_class.new(row) }
+
+  describe "#initialize" do
```

Comment:
I personally don't think this is necessary unless the constructor does some very special stuff. I have not seen instance variables being tested like this. It's also not possible for a constructor to return a different object than of its own class (afaik), so no need to test it.

Reply:
Sure its just boilerplate...